pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[tool.poetry]
+name = "pixivpy3"
+version = "3.7.2"
+description = "Pixiv API for Python (with 6.x AppAPI supported)"
+authors = ["upbit <rmusique@gmail.com>"]
+license = "Unlicense"
+readme = "README.md"
+keywords = [
+    "pixiv",
+    "api",
+    "pixivapi"
+]
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+homepage = "https://github.com/upbit/pixivpy"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+cloudscraper = "^1.2.58"
+requests = "^2.27.1"
+requests-toolbelt = "^0.10.1"
+typing-extensions = "^4.1.1"
+
+
+[tool.poetry.group.dev.dependencies]
+black = {version = ">=22.1.0", extras = ["jupyter"]}
+ruff = "^0.0.263"
+isort = ">=5.11.5"
+mypy = ">=0.931"
+pre-commit = ">=2.18.1"
+pytest = ">=7.3"
+types-requests = ">=2.27.10"
+
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.3.1"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.pytest]
```

Comment:
Not to be pendantic, but i've seen pytest's authors recommend [to use `[tool.pytest.ini_options]` instead of `[tool.pytest]`](https://docs.pytest.org/en/latest/reference/customize.html#pyproject-toml). Because of:
>the pytest team intends to fully utilize the rich TOML data format for configuration in the future, reserving the [tool.pytest] table for that. The ini_options table is being used, for now, as a bridge between the existing .ini configuration system and the future configuration format.

If I'm mistaken and we should keep it as is, please let me know.

Reply:
I'll do that then

Reply:
I combined the changes with some more improvements to the pre-commit hooks. Especially the `check-vcs-permalinks` check found broken links, which is lovely. Also, we beautify toml files now for consistency in our `pyproject.toml`